tests/Feature/Documents/DocumentsTest.php

```
@@ -0,0 +1,938 @@
+        $response->assertHeader('filename', $document->original_filename);
+    }
+
+    #[Test]
+    public function it_assignee_can_view_their_task_document(): void
+    {
+        /* Arrange */
+        $role = $this->owner->roles()->first() ?? \App\Models\Role::query()->firstOrCreate(['name' => 'owner']);
+        if ( ! $this->owner->hasRole($role->name)) {
+            $this->owner->attachRole($role);
+        }
+        $permissionName = PermissionName::DOCUMENT_VIEW instanceof PermissionName ? PermissionName::DOCUMENT_VIEW->value : PermissionName::DOCUMENT_VIEW;
```

PHPStan (2.2.5)

[warning] Instanceof between App\Enums\PermissionName::DOCUMENT_VIEW and App\Enums\PermissionName will always evaluate to true.


(instanceof.alwaysTrue)